### ISSUE TITLE
open_browser=False still opens browser

### ISSUE THREAD

**Issue by kylebarron** [kylebarron/keplergl-quickvis#2]
```py
vis = Vis()
vis.add_data(gdf)
html_path = vis.render(open_browser=False, read_only=False)
```

**Comment by ikespand** [comment]
This is due to the "open_browser" argument has not been used to avoid opening. You can edit line 149 in *keplergl_cli/keplergl_cli/keplergl_cli.py* as follow to avoid this behaviour:
```
if open_browser = True:
	webbrowser.open_new_tab('file://' + path)
```

**Comment by kylebarron** [comment]
Thanks. I just haven't gotten around to fixing this because I always use `open_browser=True`

**Comment by kylebarron** [comment]
Fixed in 6569f684e98a86624ac5e17ac0084eef9d780114
